Remotion/Core/Core/Utilities/StringUtility.cs

```diff
@@ -383,7 +384,9 @@ public static object Parse (Type type, string value, IFormatProvider formatProvi
       if (underlyingType == typeof (string))
         return value;
       if (underlyingType.IsArray)
+#nullable disable

```

Comment:
why needed?

Reply:
possible bug this method throws an ArgumentNullException if type is an array and the value is null

Reply:
yes, bug. TODO comment with jira ticket, please.